docs/wiki/protocol/architecture.md

```diff
@@ -0,0 +1,100 @@
+<h2>What is the Ethereum ?</h2>
+<p>Ethereum is an open source, globally, public and decentralized distributed ledger system.</p>
+
+<h2>What is the Blockchain?</h2>
+<p>Blockchain is a shared, immutable ledger that facilitates the process of recording transactions and tracking assets
+  in a public/private network. Blockchain is ideal for delivering that information because it provides immediate, shared and completely 
+transparent information stored on an immutable ledger.</p>
+<p>The image is a simplified and high-level overview of the Ethereum blockchain's components. 
+  This is also the hierarchical conceptualization of the Ethereum blockchain's parts.</p>
+
+![2024-02-23 15 37 18 veridelisi substack com 8c2064c68019](https://github.com/veridelisi/protocol-studies/assets/72710290/20df9189-a350-4737-abd6-bd243e4c466a)
+
+<p>The Ethereum blockchain can be conceptualized with distinct layers, each with its own role and purpose in the functioning of the network.</p>
+
+![2024-02-23 15 38 25 veridelisi substack com 740d9c04c6c5](https://github.com/veridelisi/protocol-studies/assets/72710290/ce8d9e2e-a416-41f0-b0ab-f318efded446)

```

Comment:
Again, this is unnecessary use of an image. You can utilize excalidraw to produce visualizations explaining the protocol structure like in week 1 presentation.